### DISCUSSION TITLE
Next/ Image Error

### DISCUSSION THREAD

**Discussion by aly8i** [vercel/community#992] [Help] [answered]
The Next Image is requesting a width property even tho I have set the layout="fill" objectFit="Cover", it used to work normally it suddenly crashed!!!!!!!

here is the Image:
`data.map((image,id) => {
          return (
            <SwiperSlide key={id} className={styles.wrapper}>
              <div className={styles.imgContainer}>
                <Image src={image} alt={image} layout="fill" objectFit="contain" objectPosition="center center" />
              </div>
            </SwiperSlide>
          );
        })`
here is the error:
`Image with src "/img/featured.png" is missing required "width" property.`

here is the imgContainer class:
`.imgContainer {
  width: 100vw;
  overflow: hidden;
  height: 100%;
  position: relative;
}`
the parent is position relative, what is wrong?

**Comment by amyegan** [comment, marked as answer]
Hi @aly8i! What version of Next.js do you have set in your `package.json`?

If you can share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), I can try to help with debugging.

**Reply by amyegan** [reply]
Thanks for sharing your update @aly8i! I needed to know the Next.js version to know which version of the [documentation](https://nextjs.org/docs/api-reference/next/image) was relevant. I don't know your project like you do, so I usually ask for a reproducible example to help me get familiar with your code and to see the error in real time. In this case, I don't think we actually need one.

There were some updates to [next/image](https://nextjs.org/blog/next-13#nextimage) which seem related to your problem. The [`fill`](https://nextjs.org/docs/api-reference/next/image#fill) prop is required, rather than `layout="fill"`, to skip setting `height` and `width` values for [image sizing](https://nextjs.org/docs/basic-features/image-optimization#image-sizing). The [image style](https://nextjs.org/docs/basic-features/image-optimization#styling) documentation also says that the **parent** element must have `position: relative` and `display: block` when using `fill`. Providing a [sizes](https://nextjs.org/docs/api-reference/next/image#sizes) value is recommended for improved performance when using `fill`. I hope that helps!

**Reply by aly8i** [reply]
it worked, thank you

**Comment by aly8i** [comment]
@amyegan "next": "^13.0.0", you always ask me to share a reproducible example after sharing with you the errors and the code giving the error. I finished my project 1 month before! and all my hard work seems useless because I can't deploy my application! I gave you guys my issues and you aren't providing me with a solution. My errors aren't related to the code... they are caused by modules, frameworks, dependencies, and versions... I need guidance or my 4 month's work is going to the trash.
